src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -89,4 +90,52 @@ Here are the available checks alongside the insight they offer:
  - **MessageFastViewProjection**: Follows MessageFastViewProjection cache miss rates and warns if it is below 10%. If this 
  projection is missing, this results in performance issues for JMAP GetMessages list requests. WebAdmin offers a
  [global](manage-webadmin.html#recomputing-global-jmap-fast-message-view-projection) and 
- [per user](manage-webadmin.html#recomputing-user-jmap-fast-message-view-projection) projection re-computation.
\ No newline at end of file
+ [per user](manage-webadmin.html#recomputing-user-jmap-fast-message-view-projection) projection re-computation.
+
+## Mail Processing
+
+Mail processing allows to take asynchronously business decisions on received emails.
+
+Here are it's components:
+
+ - The `spooler` takes mail out of the mailQueue and executes mail processing within the `mailet container`.
+ - The `mailet container` synchronously execute the user defined logic. This 'logic' is written though the use of `mailet`,
+ `matcher` and `processor`.
+ - A `mailet` represent an action: mail modification, envelop modification, a side effect, or stop processing.
+ - A `matcher` represent a condition to execute a mailet.
+ - A `processor` is a flow of pair of `matcher` and `mailet` executed sequentially. The `ToProcessor` mailet is a `goto` 
+ instruction to start executing another `processor`
+ - A `mail repository` allow storage of a mail as part of its processing. Standard configuration relies on the following 
+ mail repository:
+     - `cassandra://var/mail/error/` : unexpected errors that occurred during mail processing. Emails impacted by performance 
+     related exceptions, or logical bug within James code are typically stored here. These mail could be reprocessed once the 
+     cause of the error is fixed. The `Mail::error` field can help diagnose the issue. Correlation with logs can be achieved
+     via the use of the `Mail::name` field.
```

Comment:
why this `::` syntax?

Reply:
Design a field within the JSON mail object

Reply:
Mail.name is not more common?